src/Features/CSharp/Portable/Completion/CompletionProviders/TupleNameCompletionProvider.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
 {
-    internal class TupleNameCompletionProvider : CommonCompletionProvider
+    internal class TupleNameCompletionProvider : CommonCompletionProvider, IFeaturesCustomCommitCompletionProvider
```

Comment:
Why?

Reply:
It seems we will remove IFeaturesCustomCommitCompletionProvider. The "custom" commit should be a regular commit. All providers should be able to execute it. [https://github.com/dotnet/roslyn/issues/30276](https://github.com/dotnet/roslyn/issues/30276) 

---
In reply to: [222523968](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523968) [](ancestors = 222523968)